bin/wazo-service

```diff
@@ -15,44 +15,44 @@ wazo_disabled_file="/var/lib/wazo/disabled"
 
 usage() {
     cat <<-EOF
-	usage : $0 action [services_filter]
-	availables actions :
+	usage : $0 action [service_filters]
```

Comment:
Since it's "one" filter that impact "many" services, is it not more intuitive to add `s` to services instead of filter?

Reply:
English is weird... it's a "service filter" that will "filter services". It's like `service_list` is a list of services, but you wouldn't write "services_list".

Reply:
Moved to: https://github.com/wazo-platform/wazo-service/pull/26

Reply:
And I removed them here as I had to make changes to the commits anyways.